Version2/index.js

```diff
@@ -0,0 +1,12 @@
+function switchvisibility(classname) {
+    test = document.getElementsByClassName(classname);
+
+    for(const cell of test) {
+        if(cell.classList.contains("hidden")) {
```

Comment:
```suggestion
        if (cell.classList.contains("hidden")) {
```
And other occurrences as well